apps/emqx_bridge_kinesis/src/emqx_bridge_kinesis.erl

```diff
@@ -153,6 +212,103 @@ conn_bridge_examples(Method) ->
         }
     ].
 
+connector_examples(Method) ->
+    [
+        #{
+            <<"kinesis">> => #{
+                summary => <<"Kinesis Connector">>,
+                value => values({Method, connector})
+            }
+        }
+    ].
+
+bridge_v2_examples(Method) ->
+    [
+        #{
+            <<"kinesis">> => #{
+                summary => <<"Kinesis Action">>,
+                value => values({Method, bridge_v2_producer})
```

Comment:
Maybe we could use `emqx_bridge_v2_schema:action_values` to simplify this?

Reply:
Thanks for pointing it out. Did not know it existed.

Reply:
Fixed. Thanks for making these helper functions. Getting these example values right was very messy before. I think we can make it even easier by creating behaviors with callbacks functions that provide only the required information.

Reply:
I think the credit goes to @sstrigler , actually :smile_cat: